blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/BlueOceanDisplayURLImpl.java

```diff
@@ -0,0 +1,85 @@
+package io.jenkins.blueocean;
+
+import hudson.Extension;
+import hudson.model.AbstractBuild;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.tasks.test.TestResult;
+import io.jenkins.blueocean.commons.ServiceException;
+import jenkins.branch.MultiBranchProject;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+/**
+ * Created by ivan on 13/09/16.
+ */
+@Extension
+public class BlueOceanDisplayURLImpl extends DisplayURLProvider {
+    @Override
+    public String getRoot() {
+        Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins == null) {
```

Comment:
I don't think this check is required. `Jenkins.getInstance()` is `@Nonnull` return.